charts/fpl-case-service/values.aat.template.yaml

```diff
@@ -34,4 +39,6 @@ java:
         - local-authority-fallback-inbox
         - system-update-user-username
         - system-update-user-password
+        - robotics-notification-sender
+        - robotics-notification-recipient
```

Comment:
Please update fpl-case-service-configuratin (repo) with new secrets, I forgot about that step in some of other prs!